proto/emissions/events.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package zetachain.zetacore.emissions;
+
```

Comment:
Some proto file have `import "gogoproto/gogo.proto";` and some don't. When should we include and when we shouldn't